base/multidimensional.jl

```diff
@@ -1566,9 +1566,13 @@ _unique_dims(A::AbstractArray, dims::Colon) = invoke(unique, Tuple{Any}, A)
 end
 
 """
-    extrema(A::AbstractArray; dims) -> Array{Tuple}
+    extrema([f,] A::AbstractArray; dims, [init]) -> Array{Tuple}
 
-Compute the minimum and maximum elements of an array over the given dimensions.
+Compute the minimum and maximum of `f` applied to each element (if given) in the given
+dimensions of `A`.
```

Comment:
This sentence is weird when `f` is not given. Maybe this?
```suggestion
 Compute the minimum and maximum elements of `A` over dimensions `dims`.
 If `f` is provided, return the minimum and maximum elements after applying `f` to them.
```